packages/app/src/specs/generators/component/ComponentGenerator.tsx

```diff
@@ -0,0 +1,21 @@
+import { filters } from '../GeneratorsCommon'
+import ComponentGeneratorStepOne from './ComponentGeneratorStepOne.vue'
+import type { SpecGenerator } from '../types'
+import ComponentGeneratorCard from './ComponentGeneratorCard.vue'
+
+export const ComponentGenerator: SpecGenerator = {
+  card: ComponentGeneratorCard,
+  entry: ComponentGeneratorStepOne,
+  disabled: () => { },
+  show: (currentProject) => {
+    const specPattern = currentProject?.config.find((item) => item.field === 'specPattern')
+
+    if (specPattern && specPattern.from !== 'default') {
+      return false
```

Comment:
This won't work with custom spec patterns right now so don't show this card if we don't have the default spec pattern. I'm going to fix this with #23071